tests/src/JsonPatchTest.php

```diff
@@ -133,18 +138,21 @@ public function provideInvalidFieldType()
         return [
             '"op" invalid type' => [
                 (object)array('op' => array('foo' => 'bar'), 'path' => '/123', 'value' => 'test'),
-                Exception::class,
```

Comment:
I didn't realize while reviewing #7 that this only checked `Exception::class` and not the new specific kind :facepalm:

Reply:
I know! I clearly didn't look hard enough either :see_no_evil: